syntax = "proto3";

package ErrorCode;

enum EnumErrorCode
{
	//通用错误码
	CODE_SUCCESS = 0;        //成功
	//通信错误码
	CODE_PARAMETER_NULL = 1; //参数为空
	CODE_PARAMETER_INVALID = 2;  //参数不合法
    CODE_WAIT_RESP_TIMEOUT = 3; //等待response超时
	CODE_SYSTEM_ERROR = 4;
	
	//登录错误
	CODE_ACOCUNT_EMPTY = 7;
	CODE_LOGIN_ERROR = 8;
	
	//大厅错误
	CODE_CHECK_GAME_TOKEN_FAILED = 1000;
	CODE_SESSION_HAV_NO_USER = 1001;

	//房间错误
	CODE_INVALID_ROOMID = 2001;
};


